README.md

```diff
@@ -134,7 +134,7 @@ $request_body = json_decode('{
 }');
 
 $apiKey = getenv('SENDGRID_API_KEY');
-$sg = new \SendGrid($apiKey);
+$sg = new \SendGrid\SendGrid($apiKey);
```

Comment:
This does not feel right to me. I'm not sure what the right answer is, but some thoughts:

\SendGrid\APIClient
\SendGrid\Client

What do you think?